packages/components/psammead-sitewide-links/src/index.stories.jsx

```diff
@@ -24,6 +24,12 @@ storiesOf('Components|SitewideLinks', module)
     ({ text, service }) => (
       <SitewideLinks
         links={links(text, service)}
+        trustProjectLink={
+          boolean('trustProjectLink', false) && {
```

Comment:
After removing the type `boolean`, you can create two separate stories here. One with the `trustProjectLink` value being a link object,
```
        {
            href: 'https://www.bbc.co.uk/news',
            text: 'Why you can trust the bbc',
          }
```
 and one with `trustProjectLink` being undefined.

Reply:
resolved